beluga-aws-sdk/examples/tunnel.rs

```diff
@@ -6,27 +6,29 @@ use tracing::Level;
 async fn main() -> anyhow::Result<()> {
     tracing_subscriber::fmt().with_max_level(Level::INFO).init();
 
+    // let client = MqttClientBuilder::new()
+    //     .ca(include_bytes!("../../AmazonRootCA1.pem"))
+    //     .certificate(include_bytes!("../../certificate.pem.crt"))
+    //     .private_key(include_bytes!("../../private.pem.key"))
+    //     .endpoint(include_str!("../../endpoint.in"))
+    //     .thing_name(include_str!("../../thing-name.in"))
+    //     .build()?;
+
+    // for Github actions
     let client = MqttClientBuilder::new()
-        .ca(include_bytes!("../../AmazonRootCA1.pem"))
-        .certificate(include_bytes!("../../certificate.pem.crt"))
-        .private_key(include_bytes!("../../private.pem.key"))
-        .endpoint(include_str!("../../endpoint.in"))
-        .thing_name(include_str!("../../thing-name.in"))
+        .ca(&[])
+        .certificate(&[])
+        .private_key(&[])
+        .endpoint("")
+        .thing_name("")
         .build()?;
```

Comment:
We could just load these at runtime using `std::fs::read` and `std::fs::read_to_string`?